arclightning_frontend/games.html

```diff
@@ -27,41 +40,165 @@
-                    {name: "Touhou", description: "bullet hell with waifus", genre: ["bullet hell", "anime"], thumbnail_path: 'images/touhouWaifu.png', exe_path: 'C:\\Users\\THISUSER\\TOUHOU_PATH'},
-                    {name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH'},
-                    {name: "Super Smash Bros", description: "crossover fighting game", genre: ["fighter", "crossover"], thumbnail_path: 'images/peach.png', exe_path: 'C:\\Users\\THISUSER\\SMASHBROS_PATH'},
-                    {name: "Melty Blood", description: "fighter with waifus", genre: ["fighter", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg', exe_path: 'C:\\Users\\THISUSER\\MELTYBLOOD_PATH'},
-                    {name: "Touhou", description: "bullet hell with waifus", genre: ["bullet hell", "anime"], thumbnail_path: 'images/touhouWaifu.png', exe_path: 'C:\\Users\\THISUSER\\TOUHOU_PATH'},
+		selectedIndex: "touhou" ,
+                items: {
+                    'smash' : {name: "Super Smash Bros", description: "crossover fighting game", genre: ["agenre", "crossover"], thumbnail_path: 'images/peach.png'},
+                    'touhou' : {name: "Touhou", description: "bullet hell with waifus", genre: ["zgenre", "anime"], thumbnail_path: 'images/touhouWaifu.png'},
+                    'meltyblood' : {name: "Melty Blood", description: "fighter with waifus", genre: ["ygenre", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg'},
+                   'smash2' : {name: "Super Smash Bros", description: "crossover fighting game", genre: ["dgenre", "crossover"], thumbnail_path: 'images/peach.png'},
+                    'meltyBlood2' : {name: "Melty Blood", description: "fighter with waifus", genre: ["agenre", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg'},
+                   'touhou2' : {name: "Touhou", description: "bullet hell with waifus", genre: ["ygenre", "anime"], thumbnail_path: 'images/touhouWaifu.png'},
+                   'meltyBlood3' : {name: "Melty Blood", description: "fighter with waifus", genre: ["cgenre", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg'},
+                    'smash3' : {name: "Super Smash Bros", description: "crossover fighting game", genre: ["agenre", "crossover"], thumbnail_path: 'images/peach.png'},
+                    'meltyBlood3' :{name: "Melty Blood", description: "fighter with waifus", genre: ["zgenre", "anime", "2d"], thumbnail_path: 'images/meltyBloodWaifu.jpg'},
+                   'touhou3' : {name: "Touhou", description: "bullet hell with waifus", genre: ["bgenre", "anime"], thumbnail_path: 'images/touhouWaifu.png'},
+	}
+            },
+		
+	computed:{
```

Comment:
I think you can get rid of this
https://github.com/MTSUMakers/ArcLightning/blob/916044c76b374c2fb9d27bba4ecc9364725fd0ef/arclightning_frontend/games.html#L77-L93

Reply:
yes you are so efficient